CONTRIBUTING.md

```diff
@@ -31,3 +32,52 @@ The most important thing to include in your pull request are *tests* - please wr
 Starting December 1, 2017, all contributions will be assumed to be released under a dual license - the [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0) and the [3-Clause BSD License](https://opensource.org/licenses/BSD-3-Clause) unless otherwise specified in the pull request.
 
 All contributions before December 1, 2017 except those explicitly relicensed, are only under the 3-clause BSD license.
+
+## Building and releasing
+
+When you get the source, it does not contain the internal zoneinfo
+database. To get (and update) the database, run the updatezinfo.py script. Make sure
+that the zic command is in your path, and that you have network connectivity
+to get the latest timezone information from IANA, or from [our mirror of the
+IANA database](https://dateutil.github.io/tzdata/).
+
+Starting with version 2.4.1, all source and binary distributions will be signed
+by a PGP key that has, at the very least, been signed by the key which made the
+previous release. A table of release signing keys can be found below:
+
+Releases | Signing key fingerprint
```

Comment:
The release key stays in the README, but don't worry about doing that, I'll just rearrange it before merging.